### PR TITLE
Alexandr samohvalov second java task

### PR DESCRIPTION
**Pull request by LLLAlexandrLL**

### CODE REVIEW

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package alexandr_samohvalov;
+
+public class NumbersMain {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java:3]: why this class is empty?

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]
Will you create any additiion task?
In task 2 are more points to resolve.

**Comment by LLLAlexandrLL** [comment]
New commits added

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/ConditionsMain.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            System.out.println("Number is positive");
+        } else if (a < 0) {
+            System.out.println("Number is negative");
+        } else {
+            System.out.println("Number is equal to zero");
+        }
+    }
+
+    //Check if a number is divisible by both 2 and 3.
+    public static void s(String[] args) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/ConditionsMain.java:20]: method name s is incompatible 
method name should dscribe what this method does

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/ConditionsMain.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
+package alexandr_samohvalov;
+
+import java.util.Scanner;
+
+public class ConditionsMain {
+    // Check if a number is positive, negative, or zero.
+    public static void maaain(String[] args) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/ConditionsMain.java:7]: same

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
+package alexandr_samohvalov;
+
+import java.util.Random;
+
+public class NumbersMain {
+    //Find the square root of a number
+    public static void ssaww(String[] args) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java:7]: same

Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java:7]: do you need String[] parameter here?

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/ConditionsMain.java (hunk 2 above)
Comment (dgrimaliuc) [reply SimpleJavaProject/src/main/java/alexandr_samohvalov/ConditionsMain.java:20]: same with paramters

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    public static void maaasfin(String[] args) {
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);
+
+        System.out.println(randomNumber);
+    }
+
+
+    // Generate a random number between 10 and 100.
+    public static void maiaafqwn(String[] args) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java:22]: same with method name and parameters

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+public class NumbersMain {
+    //Find the square root of a number
+    public static void ssaww(String[] args) {
+        int a = 49;
+        System.out.println(Math.sqrt(a));
+    }
+
+    // Generate a random number between 0 and 100.
+    public static void maaasfin(String[] args) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java:13]: please review all methods and fix method name and unused parameter indicated previously

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/StringsMain.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    //  Check if a string contains a specific substring.
+    public static void aaq(String[] args) {
+        String txt = "I hope I am right ";
+        System.out.println(txt.endsWith(" "));
+    }
+
+
+    //  Remove all whitespace characters from a string.
+    public static void aafsmain(String[] args) {
+        String txt = "    I hope I am right     ";
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/StringsMain.java:26]: try to use  \t instead of multiple spaces

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]
Please update your branch, 
pull last changes from master or do rebase.

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/StringsMain.java [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+
+    //  Remove all whitespace characters from a string.
+    public static void RemoveWhiteSpace(String[] RemoveWhiteSpace) {
+        String txt = "    I hope I am right     ";
+        System.out.println(txt.trim());
+    }
+
+    //  Extract a substring from a string.
+    public static void ExtractSubStrFromStr(String[] ExtractSubStrFromStr) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/StringsMain.java:31]: the name of methods should start from lowercase letter
change ExtractSubStrFromStr to extractSubStrFromStr

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/StringsMain.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+public class StringsMain {
+    // Count the number of characters in a given string.
+    public static void CountNumberChar(String[] CountNumberChar) {
+        String txt = "I hope I am right ";
+        System.out.println(txt.length());
+    }
+
+
+    // Convert a string to uppercase.
+    public static void ConvertStrUppercase(String[] ConvertStrUppercase) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/StringsMain.java:12]: same

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/StringsMain.java [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+        String txt = "I hope I am right ";
+        System.out.println(txt.toUpperCase());
+    }
+
+    //  Check if a string contains a specific substring.
+    public static void StrContainsSpecificSubStr(String[] StrContainsSpecificSubStr) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/StringsMain.java:18]: same

File: SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+        int b = 6;
+        int c = 9;
+        int sum = a + b + c;
+        int average = sum / 3;
+
+        System.out.println(average);
+    }
+
+    // Find the maximum of 4 numbers.
+    public static void FindMaxNum(String[] FindMaxNum) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java:41]: same

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]
rebase on master and remove SimplJavaProject folder

**Review by dgrimaliuc-crunchyroll — CHANGES_REQUESTED** [review]
you didn't removed SimpleJavaProject folder

**Pull request closed without merging**